samples/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.springframework.cloud</groupId>
+	<artifactId>spring-cloud-stream-samples-parent</artifactId>
+	<version>4.0.0-SNAPSHOT</version>
+	<name>spring-cloud-stream-samples</name>
+	<description>spring-cloud-stream-samples</description>
+	<packaging>pom</packaging>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.0.0-SNAPSHOT</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+
+	<properties>
+		<avro.version>1.11.0</avro.version>
+		<confluent.version>7.0.1</confluent.version>
+		<spring-cloud.version>2022.0.0-SNAPSHOT</spring-cloud.version>
+	</properties>
+
+
+	<modules>
+		<module>avro-samples</module>
+	</modules>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dependencies</artifactId>
```

Comment:
This is a circular dependency

Reply:
@spencergibb can you clarify - I don't see if from 1st glance.

Reply:
Spring cloud dependencies depends on spring cloud stream and now stream depends on spring cloud dependencies. This will have to look different otherwise releases will fail.

Reply:
we can use `spring-cloud-stream-dependencies` instead. During a release when the samples build runs, we already have the `spring-cloud-stream-dependencies` module built, thus I think solves the circular dependency from spring cloud dependencies.